site/src/components/series/point.md

```diff
@@ -47,18 +81,60 @@ Which gives the following:
     var f = createFixture('#series_point', desiredWidth, desiredHeight, null, function() { return true; });
     var container = f.container, data = f.data,
       xScale = f.xScale, yScale = f.yScale;
-    {{{example2-code }}}
+    {{{example-code-1}}}
 }());
 </script>
 
-The series assumes that each object has a `date` property which defines its x-location, and a `close` property which defines its y-location. However, these can be changed via the `xValue` and `yValue` properties.
+The series assumes that each object has a `date` property which defines its x-location, and a `close` property which 
+defines its y-location. However, these can be changed via the `xValue` and `yValue` properties.
 
-You can supply a function via the `radius` property that specifies the radius for each point. You can also modify how the series is rendered via the `decorate` function which is supplied the selection containing the `g` element that is the parent for each `circle`:
+You can supply a function via the `size` property that specifies the area size for each point. You can also modify how 
+the series is rendered via the `decorate` function which is supplied the selection containing the `g` element that is 
+the parent for each `path`:
 
 ```js
-{{{example-code}}}
+{{{example-code-2}}}
 ```
 
 Which results in the following (slightly odd looking) series:
 
-{{>example-fixture}}
+<div id="series_point_2" class="chart"> </div>
+<script type="text/javascript">
+(function() {
+    var desiredWidth = $('#series_point_2').width(),
+        desiredHeight = desiredWidth / 2.4; //keeps the width-height ratio at 600-250 (defaults for createFixture)
+    var f = createFixture('#series_point_2', desiredWidth, desiredHeight, null, function() { return true; });
+    var container = f.container, data = f.data,
+      xScale = f.xScale, yScale = f.yScale;
+    {{{example-code-2}}}
+}());
+</script>
+
+Thanks to the series element being built around the `d3.svg.symbol`, the following symbol `types` are also supported:
```

Comment:
As the series is built using the `d3.svg.symbol`, the following symbol `types` are also supported: